resources/sql/queries.sql

```diff
@@ -3,7 +3,7 @@
 -- - Get catalogue items
 -- - :items vector of item ids
 -- - :resource resource id to fetch items for
-SELECT ci.id, ci.title, res.resid, ci.wfid, ci.formid, ci.start, ci.endt, ci.enabled, ci.archived
+SELECT ci.id, ci.title, res.resid, ci.wfid, ci.formid, ci.start, ci.endt as "end", ci.enabled, ci.archived
```

Comment:
This does not clean up the database however? Was it your intention?

Reply:
It's an old field. Better to rename it physically in a separate PR. It smells like an invasive operation.

Reply:
Yes that's why I suggested a separate task for it.